static/modules/gamemanager.js

```diff
@@ -71,6 +71,8 @@ class GameManager {
   /** @type {boolean} */
   gamePause = false;
 
+  displayCanvas = undefined;
+
```

Comment:
Why did this get added?

Edit: This also has literally no references but one in a function that also has literally no references itself, so get rid of it.